admin/commands/management/add.go

```diff
@@ -97,5 +108,14 @@ func processGlobalAddFlagsWithSocket(cmd connectionGetter, opts AddCommonFlags)
 		portI = int(opts.AddPortFlag)
 	}
 
-	return serviceName, socket, host, uint16(portI), nil
+	if opts.AddExposeExporterAddressFlag {
+		exposeExporterAddress = opts.AddExposeExporterAddressFlag
+	}
```

Comment:
I think it can be simplified by removing `exposeExporterAddress` and using `opts.AddExposeExporterAddressFlag` directly

Reply:
Can we use `opts.AddExposeExporterAddressFlag` in the place we call it without putting it into another struct?

Reply:
> Can we use `opts.AddExposeExporterAddressFlag` in the place we call it without putting it into another struct?

Do you mean without the `globalAddServiceParams` struct? In that case the function will return five values - I feel that's a lot and hard to keep track of their meanings (considering we don't use named returns)